basket/news/api.py

```diff
@@ -35,12 +39,50 @@
 def list_newsletters(request):
     # Get the newsletters as a dictionary of dictionaries that are
     # easily jsonified
-    result = {}
-    for newsletter in Newsletter.objects.all():
-        newsletter.languages = newsletter.languages.split(",")
-        result[newsletter.slug] = NewsletterSchema.from_orm(newsletter).dict()
+    newsletters = {n.slug: NewsletterModelSchema.from_orm(n).dict() for n in Newsletter.objects.all()}
+    return {"status": "ok", "newsletters": newsletters}
+
+
+### /api/v1/users URLS
+
+user_router = Router()
+
+
+@user_router.post(
+    "/recover/",
+    url_name="users.recover",
+    description="Send recovery email",
+    response={
+        200: OkSchema,
+        400: ErrorSchema,
+        404: ErrorSchema,
+        500: ErrorSchema,
+    },
+)
+def recover_user(request, email: Form[EmailStr]):
```

Comment:
I'm debating moving to a JSON only API. Bedrock should be able to migrate to that easily enough but I'm curious how far this is used outside of Bedrock. It may mean keeping the old path around.

Reply:
I wouldn't think this is used outside of bedrock. JSON only +1 from me.

Reply:
Same here - JSON all the way